app/middleware/setLocals.js

```diff
@@ -1,22 +1,27 @@
 const notFound = require('../middleware/renderer').notFound;
 
+function hostNameAndProtocol(req) {
+  return `https://${req.hostname}`;
```

Comment:
Ideally yes but the SSL termination is done at the load balancer so the request coming through to the application is `http`. Which would be fine, however, non-secure requests are redirected to `https` so we get into an infinite loop or requesting pages over `http` but having them served over `https`.